app/Models/Signup.php

```diff
@@ -41,7 +41,7 @@ class Signup extends Model
      * @var array
      */
     public static $indexes = [
-        'campaign_id', 'campaign_run_id', 'updated_at', 'northstar_id', 'id',
+        'campaign_id', 'campaign_run_id', 'updated_at', 'northstar_id', 'id', 'quantity',
```

Comment:
Oh, hmm! Two questions here ––

1. Does this column exist on the signup anymore? I think we [decided](https://github.com/orgs/DoSomething/teams/team-bleed/discussions/10) it wouldn't.
2. If it does, we need it to be indexed!

Reply:
oh @DFurnes you're so right! I totally blanked on that! 

This column still exists in the database but when we switch to saving quantity on the post, we will no longer be storing the quantity in this column. 

This is what you were talking about re: the heavy query to order these by. Should we discuss this in the discussions you linked if this is the time we want to store this?

Reply:
I think this definitely makes sense to discuss as a team! 👍

Reply:
for documentation: we decided to store the total quantity on the signup! https://github.com/orgs/DoSomething/teams/team-bleed/discussions/10?from_comment=13#discussion-10-comment-13